Mayflower Mk II

VividPups opened the pull request:
## About the PR
The first thing I made in SS14 was a ship. After many failed attempts, I finally got a more unique ship in the game that was similar to the Sprinter.  The picture below is the MK 1  mayflower. An abandoned, overgrown trash ship that everyone seems to love. It was the cheapest, it did its job well, and it just needed a little love and care. Now its time it grows up remove the dirt and trash and be the best it really can 

Offering Extra lattice protection and reinforced haul in the front and in the engine bay. This ship is rated for about 1-6 members. It offers a lounge room and a small medical bay, a cargo area, and a free space. It has an extra large armory with an reinforce vault on the right side. There are 5 entrances/exits to this ship. 
With name doors and buttons. This bad girl comes in at 80k, which is 30k more than the Mark 1. However, instead of reinforced steel. The hull is now made out of shuttle walls instead. This is heavily armored and ready to fight. 


<img width="2539" height="1380" alt="image" src="https://github.com/user-attachments/assets/49be6006-6a05-4083-8aef-cea336f04984" />
https://github.com/new-frontiers-14/frontier-station-14/pull/653#issue-2024966179
https://github.com/new-frontiers-14/frontier-station-14/pull/734

## Why / Balance
Alot of people were sad when I removed the mayflower. It was a lot of people said it was an amazing ship, even with all the trash. However, I disagree with a lot of the changes that were going to happen to ships, and I ask to be removed. This PR adds mark two. It's clean up, it had the love, and it's ready to man the dreams and hopes of the captain and their crew
.
## How to test
In-game buy it 

## Media
<img width="709" height="1026" alt="image" src="https://github.com/user-attachments/assets/c0c867f9-7f50-4190-9cec-3c803c086cec" />
<img width="967" height="227" alt="image" src="https://github.com/user-attachments/assets/9d8b033f-bec5-426b-8f8c-98c59e69d5b2" />
<img width="713" height="954" alt="image" src="https://github.com/user-attachments/assets/eb9ca2bf-acda-4d41-8340-f48e0a09eb7c" />
<img width="1163" height="879" alt="image" src="https://github.com/user-attachments/assets/62f48742-f1dd-4454-8a8f-e48630ab31b2" />
<img width="1048" height="728" alt="image" src="https://github.com/user-attachments/assets/1111de61-2dbf-4e89-a06a-98794a4698c3" />


ALSO  shower gun meme 
<img width="703" height="428" alt="image" src="https://github.com/user-attachments/assets/381db561-3d16-4751-9a91-963a5059b24d" />

## Addition note 
I hate the Atmos I am going to rework it when pipe stacking is a thing 
## Breaking changes
your bank account when you see how much you love this ship 

**Changelog**

- add: Added Expedition Ship Mayflower mk II

BlueNexus commented:
I'll handle this, as the resident shuttle reviewer :3

BlueNexus commented:
> Blu I was already requested to put the mark 1 mayflower in the game as well

I would not approve it in its current state, except as perhaps a scrapyard ship. let's focus on the mk2 in this pr

VividPups commented:
> So, what is this ship supposed to _do_, exactly?

It's a bounty hunter ship like the Sprinter

BlueNexus commented:
> > > Blu I was already requested to put the mark 1 mayflower in the game as well
> > 
> > 
> > I would not approve it in its current state, except as perhaps a scrapyard ship. let's focus on the mk2 in this pr
> 
> also it was orginially in the scrapyard and was the only Expedition capable scrapyard ship

I wouldn't approve it as a new scrapyard ship, fwiw

VividPups commented:
> > > > Blu I was already requested to put the mark 1 mayflower in the game as well
> > > 
> > > 
> > > I would not approve it in its current state, except as perhaps a scrapyard ship. let's focus on the mk2 in this pr
> > 
> > 
> > also it was orginially in the scrapyard and was the only Expedition capable scrapyard ship
> 
> I wouldn't approve it as a new scrapyard ship, fwiw

:/

BlueNexus commented:
Sorry for the delay, reviewing this tomorrow

BlueNexus commented:
Reviewing now, please hold

BlueNexus reviewed (changes requested):
Reviewed on discord, see feedback

VividPups commented:
<img width="1272" height="88" alt="image" src="https://github.com/user-attachments/assets/2851f6f8-aec8-4032-b44d-2f6b68b8be17" />
New appearisal cost 

<img width="809" height="644" alt="image" src="https://github.com/user-attachments/assets/1fc37357-f570-4d4c-aaeb-4eec59a3891a" />
<img width="568" height="484" alt="image" src="https://github.com/user-attachments/assets/441131c8-c1c1-40f7-b71b-7a7cfa9afae3" />
Add two starting Merc hardsuits 

<img width="400" height="50" alt="image" src="https://github.com/user-attachments/assets/57bfdae0-838f-4a89-85ed-802611363181" />
Add lockdown buttons to the entrances (a QoL thing)

<img width="535" height="320" alt="image" src="https://github.com/user-attachments/assets/3f170023-f3d0-4efa-89b5-9680a03bfac2" />
Add a reclaimer, an ore processor, and a Scrap processor as a flat pack item so people can place them where they choose  
 
 
<img width="496" height="593" alt="image" src="https://github.com/user-attachments/assets/da18791b-3d2b-45f4-afc3-d9597f5fac43" />
d
<img width="582" height="470" alt="image" src="https://github.com/user-attachments/assets/14b9f7d6-7b86-4cb4-b125-32f3ac5616d6" />
<img width="549" height="610" alt="image" src="https://github.com/user-attachments/assets/18e6f0b9-3f5e-4532-98d5-4a94cc1d0124" />
<img width="574" height="561" alt="image" src="https://github.com/user-attachments/assets/60284cac-c6d6-4179-80dd-c148b6086eed" />
adjusted` the amount of forward thrusters and thruster location 

<img width="537" height="409" alt="image" src="https://github.com/user-attachments/assets/a3b7a326-3a78-46c7-9f64-545a9b28d675" />
Added an Undone telecom server (they can spend the money to put a key in) 

<img width="888" height="273" alt="image" src="https://github.com/user-attachments/assets/5045e762-c20b-4a52-8f1a-ebd408c6476d" />
slight revamp of the bottom to allow easier cargo access

VividPups commented:
<img width="714" height="1007" alt="image" src="https://github.com/user-attachments/assets/1b335238-de6c-437d-89ec-41834fa3f93c" />

BlueNexus commented:
When you say 'undone', it still starts with the common and medical key, like other exped ships, right?
